database/factories/UserFactory.ts

```diff
@@ -0,0 +1,9 @@
+import User from 'App/Models/User'
+import Factory from '@ioc:Adonis/Lucid/Factory'
+
+export default Factory.define(User, ({ faker }) => {
+  return {
+    email: faker.internet.email(),
+    password: faker.internet.password(),
+  }
+}).build()
```

Comment:
Kannst du das in nem extra PR machen?

Reply:
okay

Reply:
nee warte

Reply:
Hatte erst später gesehen, dass du das ja für die Tests brauchst. Sorry :)